mock/py/mock.py

```diff
@@ -453,6 +453,10 @@ def setup_logging(config_path, config_opts, options):
     if options.trace:
         logging.getLogger("trace").propagate = 1
 
+    logging.getLogger("mockbuild").stderr_line_prefix = ""
```

Comment:
It will probably work, though `stderr_line_prefix` isn't part of ` logging.Logger` api, and the solution thus isn't really clean.